Dockerfile

```
@@ -66,6 +66,35 @@ USER 65532:65532
 
 ENTRYPOINT ["/vault-secrets-operator"]
 
+# ubi release image
+# -----------------------------------
+FROM registry.access.redhat.com/ubi9/ubi-micro:9.2-9 as release-ubi
+
+ENV BIN_NAME=vault-secrets-operator
+ARG PRODUCT_VERSION
+ARG PRODUCT_REVISION
+ARG PRODUCT_NAME=$BIN_NAME
+# TARGETARCH and TARGETOS are set automatically when --platform is provided.
+ARG TARGETOS TARGETARCH
+
+LABEL name="Vault Secrets Operator" \
+      maintainer="Team Vault <vault@hashicorp.com>" \
```

I'm not sure if vault@ is a monitored inbox?

```suggestion
      maintainer="Team Vault <team-vault@hashicorp.com>" \
```

---

Hmm, I don't know for sure either, but `vault@hashicorp.com` is what we've been using in our other Dockerfile's, e.g.
* https://github.com/hashicorp/vault/blob/main/Dockerfile#L19
* https://github.com/hashicorp/vault/blob/main/Dockerfile#L19

---

👍 I'm fine with vault@ as well